src/tests/logout.test.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import '@testing-library/jest-dom'
+import { render } from "@testing-library/react";
+import logoutReducer from '../redux/reducers/reducerLogout'
+import { logoutAction, logoutRequest, logoutSuccess, logoutError } from '../redux/actions/authLogout';
+import { Provider } from "react-redux";
+// import { LOGOUT_REQUEST, LOGOUT_SUCCESS, LOGOUT_ERROR } from '../redux/types/logoutTypes,
+import { BrowserRouter as Router, Switch, Route, Redirect } from "react-router-dom";
+import userEvent from '@testing-library/user-event';
+import store from "../redux/store.js";
+import Logout from "../components/Logout"
+
+const LogoutComponent = () => {
+    return render(
+      <Provider store={store} >
+        <Router>
+      <Logout />
+      </Router>
+    </Provider>
+    )
+  }
+
+  describe('Test Logout', ()=>{
+   
+
+    it('Capturing Snapshot of Logout', () => {
+        const renderedValue = LogoutComponent()
+        expect(renderedValue).toMatchSnapshot();
+    });
+
+    it('should test action', ()=>{
+        expect(logoutRequest()).toHaveProperty('type')
+    })
+
+    it('should test action', ()=>{
```

Comment:
maybe you can try and give different names to these test titles, since they don't have the same assertions.